### DISCUSSION TITLE
FourCC Metadata List

### DISCUSSION THREAD

**Discussion by WURS5000** [gopro/labs#321] [Q&A] [answered]
Hi there,

i know there is a pretty good list over here:
https://gopro.github.io/labs/control/extensions/

While lookin thru metadata of some files i noticed kinda more fourCC wich are not listed but i would like to know what they stand for, especially lookin for:

OREN and CLKS

THanks in advance

**Comment by dnewman-gpsw** [comment, marked as answer]
OREN and CLKS aren't Labs settings.  Only header metadata within
 "DEVC": {
    "DVID": "USRM",
    "DVNM": "User Metadata",
    "STRM": {
       ...
     }
are settings for Labs.

OREN an CLKS are general camera metadata for internal use, like within the Quik app. Some the more usefully streaming metadata is documented here https://gopro.github.io/gpmf-parser/

OREN is orientation, "U" for Up, for the landscape vs portrait
CLKS is what was the origin of the current time date, 1 -  manual, 2 - App, 3 - GPS

**Comment by WURS5000** [comment]
Thanks, that's what i was lookin for!
